src/transformations.jl

```diff
@@ -15,38 +17,16 @@ a harmonic trap potential at position ``x``, i.e.:
             \\frac{1}{\\sqrt{2^n n!}} H_n\\left(\\frac{x}{x_0}\\right)
 ```
 """
-function transform(::Type{S}, b1::PositionBasis, b2::FockBasis; x0=1) where S
-    T = Matrix{S}(undef, length(b1), length(b2))
-    xvec = samplepoints(b1)
-    A = hermite.A(b2.N)
-    delta_x = spacing(b1)
-    c = 1.0/sqrt(x0*sqrt(pi))*sqrt(delta_x)
-    for i in 1:length(b1)
-        u = xvec[i]/x0
-        C = c*exp(-u^2/2)
-        for n=b2.offset:b2.N
-            idx = n-b2.offset+1
-            T[i,idx] = C*horner(A[n+1], u)
-        end
+function transform(::Type{S}, bp::PositionBasis, bf::FockBasis) where S
+    T = Matrix{S}(undef, length(bp), length(bf))
+    xvec = samplepoints(bp)
+    C = pi^(-1/4)*sqrt(spacing(bp))
```

Comment:
I am reluctant to remove the undocumented `x0` keyword argument. It seems it is just some sort of rescaling (presumably that is why it is called characteristic length). Could you add that functionality as well and undelete the tests?

Reply:
I've added it back in and undeleted the tests, although I didn't document the keyword since I'm not sure exactly what it's supposed to physically correspond to or why it would be needed.

Reply:
Thank you! And yes, not documenting it makes perfect sense -- no need to make it any more public until there is more serious commitment to that feature, I just wanted to make sure we do not break someone's code out there in the wild.